dark/assets/css/styles.css

```diff
@@ -1,6 +1,7 @@
 /* Edo Font */
 
 @import url('http://fonts.cdnfonts.com/css/edo');
+@import url('https://fonts.googleapis.com/css2?family=Kanit&display=swap');

```

Comment:
Create another comment called Kanit Font and put the font tag inside the comment

Reply:
Ok

Reply:
> Delete the early commits by creating a new branch, reset the branch, get a pull from the upstream, do, add, commit and push the changes.

So I would need to open a new pull request?

Reply:
@YoshithaRathnayake ?

Reply:
> > Delete the early commits by creating a new branch, reset the branch, get a pull from the upstream, do, add, commit and push the changes.
> 
> So I would need to open a new pull request?

No no now you have already created a branch called dev no so you have to reset it by `git reset --hard upstream/master` and get a pull from the upstream as `git pull upstream master` and do the changes and commit and push

Reply:
![image](https://user-images.githubusercontent.com/52712652/195989437-063ff80a-c523-4fc2-86d4-27745f7d7aa0.png)

@YoshithaRathnayake

Reply:
Ah sorry I couldn't remember one thing wait I will find that and send it to you @AlgyJr

Reply:
`git remote add upstream https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE` and `git fetch upstream` after that the changes in the above comment @AlgyJr

Reply:
![image](https://user-images.githubusercontent.com/52712652/195990299-91191f4b-a711-4540-8e9b-44c007a048a6.png)

This is the project now, how is suppost to do the changes missing files, I can't find the page file you request to change.

Reply:
@YoshithaRathnayake

Reply:
can you delete the forked folder and do this process again if you can bcs that's the only way @AlgyJr

Reply:
![image](https://user-images.githubusercontent.com/52712652/195990715-6e72a742-5900-44c0-818a-7885d973e7d7.png)

Reply:
> can you delete the forked folder and do this process again if you can bcs that's the only way @AlgyJr

Ok